src/tokens/eth/0x428548d53E889d7A0D51F4142BA53463808c1BE6.json

```diff
@@ -0,0 +1,26 @@
+{
+  "symbol": "TWS",
+  "name": "Energy27",
+  "type": "ERC20",
+  "address": "0x428548d53E889d7A0D51F4142BA53463808c1BE6",
+  "ens_address": "",
+  "decimals": 18,

```

Comment:
[Etherscan lists the token decimal as 8](https://etherscan.io/token/0x428548d53E889d7A0D51F4142BA53463808c1BE6).

Reply:
Hello, I solved this issue and i upgrade to old contract address.

TWS team decide to use old contract address.
https://etherscan.io/token/0xa4f267b2bf5c47873ec85cb55749368bc15ec2ec
contract and logo are updated.

now its fine. 

thank you gamelielhere